tests/libtest.sh

```diff
@@ -50,6 +50,19 @@ vg_exists() {
   return 1
 }
 
+# Tests if the physical volume pv_name exists
+pv_exists() {
```

Comment:
@shishir-a412ed I just implemented pv_exists() in libcss.sh. How about that implementation?